cms/djangoapps/contentstore/docs/decisions/0003-hybrid-approach-for-public-apis.rst

```diff
@@ -1,6 +1,21 @@
 0003: Hybrid approach for public course authoring APIs
 ======================================================
 
+Status
+------
+
+Rejected
+
+Reason for rejection
+--------------------
+
+The objectives for public authoring APIs changed from the time this decision was made:
+we are now implementing a limited set of experimental APIs that are intended as an
+MVP for a future, more complete set of APIs. The authoring APIs we are implementing
+are a public set of wrappers around existing functionality, and are not intended to
+be used for production course authoring yet. The responsibility for avoiding conflicts
+and resolving them if they occur is on the user.
+
```

Comment:
I would suggest the following minor change:

The objectives for public authoring APIs changed from the time this decision was made:
we are now limiting our offering to a set of experimental APIs with which to flesh our what a supported set of APIs might become. As such, the authoring APIs we are now implementing
are just a public set of wrappers around existing functionality, and are not fit for production course authoring. The responsibility for avoiding conflicts and resolving them, if they occur, is on the user.